test/components/util/util.go

```diff
@@ -40,6 +40,15 @@ import (
 	"k8s.io/client-go/transport/spdy"
 )
 
+const (
+	// RetryInterval is time for test to retry.
+	RetryInterval = time.Second * 5
+	// Timeout is time after which tests stops and fails.
+	Timeout = time.Minute * 5
+	// TimeoutSlower is time after which tests stops and fails.
+	TimeoutSlower = time.Minute * 9
```

Comment:
`Slower` sounds weird to me. I think just `Slow` might be better, so it indicates that this test case is slow, so it should get higher timeout. If we add `Slower`, we could also have `Slowest` etc. I think `Slow` is more clear.
```suggestion
	TimeoutSlow = time.Minute * 9
```